File: clientPython/zeroc-ice-3.7.2/csharp/test/Ice/objects/UnexpectedObjectExceptionTestI.cs
//
// Copyright (c) ZeroC, Inc. All rights reserved.
//

namespace Ice
{
    namespace objects
    {
        public sealed class UnexpectedObjectExceptionTestI : Ice.Blobject
        {
            public override bool ice_invoke(byte[] inParams, out byte[] outParams, Ice.Current current)
            {
                var communicator = current.adapter.getCommunicator();
                var @out = new Ice.OutputStream(communicator);
                @out.startEncapsulation(current.encoding, Ice.FormatType.DefaultFormat);
                var ae = new Test.AlsoEmpty();
                @out.writeValue(ae);
                @out.writePendingValues();
                @out.endEncapsulation();
                outParams = @out.finished();
                return true;
            }
        }
    }
}

File: clientPython/zeroc-ice-3.7.2/java-compat/test/src/main/java/test/Ice/objects/UnexpectedObjectExceptionTestI.java
//
// Copyright (c) ZeroC, Inc. All rights reserved.
//

package test.Ice.objects;

import test.Ice.objects.Test.AlsoEmpty;

public final class UnexpectedObjectExceptionTestI extends Ice.Blobject
{
    @Override
    public boolean
    ice_invoke(byte[] inParams, Ice.ByteSeqHolder outParams, Ice.Current current)
    {
        Ice.Communicator communicator = current.adapter.getCommunicator();
        Ice.OutputStream out = new Ice.OutputStream(communicator);
        out.startEncapsulation(current.encoding, Ice.FormatType.DefaultFormat);
        out.writeValue(new AlsoEmpty());
        out.writePendingValues();
        out.endEncapsulation();
        outParams.value = out.finished();
        return true;
    }
}
